src/igvfd/searches/configs/construct_library_set.py

```diff
@@ -57,6 +90,21 @@ def construct_library_set():
                     'file_set_type',
                     'selection_criteria',
                     'scope',
+                    'associated_phenotypes.term_name',
+                    'tiling_modality',
+                    'guide_type',
+                    'small_scale_gene_list.symbol',
+                    'orf_list.gene',
```

Comment:
No facet appears for `orf_list.gene`. Should it instead show `off_list.gene.symbol`?

Reply:
You are correct, fixed.